cranelift-wasm/src/code_translator.rs

```diff
@@ -917,30 +917,46 @@ pub fn translate_operator<FE: FuncEnvironment + ?Sized>(
         | Operator::TableSize { .. } => {
             return Err(WasmError::Unsupported("proposed bulk memory operators"));
         }
+        Operator::V128Const { value } => {
+            let constant = builder.func.dfg.constants.insert(value.bytes().to_vec());
+            state.push1(builder.ins().vconst(I32X4, constant)) // TODO need to decide what to do about I32X4 here; the WASM spec has V128
```

Comment:
@bnjbvr and I talked about this but probably others need to weigh in (@sunfishcode?): the SIMD spec talks about a `v128.const` instruction but we do not have anything exactly equivalent to `v128` in cranelift. Some options:
 - use a default (e.g. see `I32X4` here) and then legalize later to raw_bitcast this value to the next instruction that uses it
 - use `BVType` (see https://github.com/CraneStation/cranelift/blob/2b1cfea1634c58d22b880eb082c748350041c3f0/cranelift-codegen/meta/src/cdsl/types.rs#L395-L399)
 - use `I128` from #795 with some legalization
 - add a new `V128` type

Reply:
From talking to @sunfishcode: the current best decision is to add bitcasts from-to `I8x16`.

Reply:
This is resolved by 
d76a85e